tests/core/helpers.py

```diff
@@ -130,15 +135,19 @@ class TestModelFuture(ModelTrainerBase.ModelFutureBase):
 
         def __init__(self, parent, trained_model, save_path, save_with_id):
             super().__init__(
-                trainer_name=parent.instance_name,
-                training_id=str(uuid.uuid4()),
-                save_path=save_path,
+                future_name=parent.instance_name,
+                future_id=str(uuid.uuid4()),
```

Comment:
This is interesting. I hadn't noticed that switching to have `ModelFutureBase` inherit from `JobFutureBase` caused this kwarg naming change. I think this could _technically_ be considered an API breaking change since authors of `Trainers` may have explicitly called `training_id=` when initializing the parent. I think the right way to handle this is to add name aliases in `__init__` in the subclasses of `JobFutureBase`.

Reply:
Good call! I just updated `ModelTrainerFutureBase`